ha_lineairdb.cc

```diff
@@ -104,9 +104,9 @@
 #include "sql/sql_plugin.h"
 #include "sql/table.h"
 #include "typelib.h"
-
-#define INPLACE_UPDATE
```

Comment:
Is it deleted since it's unused?

Reply:
yes, it used to be in update_row but we have completely different implementation now so I deleted this